test/tools/unified-spec-runner/operations.ts

```diff
@@ -793,9 +817,25 @@ operations.set('runCursorCommand', async ({ entities, operation }: OperationFunc
 operations.set('createCommandCursor', async ({ entities, operation }: OperationFunctionParams) => {
   const collection = entities.getEntity('db', operation.object);
   const { command, ...opts } = operation.arguments!;
+  switch (opts.cursorType) {
+    case 'tailableAwait':
+      opts.tailable = true;
+      opts.awaitData = true;
+      break;
+    case 'tailable':
+      opts.tailable = true;
+      break;
+    default:
+      break;
+  }
+  delete opts.cursorType;
   const cursor = collection.runCursorCommand(command, {
     readPreference: ReadPreference.fromOptions({ readPreference: opts.readPreference }),
-    session: opts.session
+    session: opts.session,
+    tailable: opts.tailable,
+    awaitData: opts.awaitData,
+    timeoutMode: opts.timeoutMode,
+    timeoutMS: opts.timeoutMS
```

Comment:
passing through relevant options to ensure that runCommandCursors are properly configured